index.vwf.yaml

```diff
@@ -333,6 +333,243 @@ children:
+                this.wheelFL = this.find( "Cylinder014/Cylinder058/wheel" )[ 0 ];
+              }
+              return this.wheelFL;
+          wheelBL:
+            get: |
+              if ( !this.wheelBL ) {
+                this.wheelBL = this.find( "Cylinder014/Cylinder059/wheel001" )[ 0 ];
+              }
+              return this.wheelBL;
+          wheelFR:
+            get: |
+              if ( !this.wheelFR ) {
+                this.wheelFR = this.find( "Cylinder014/Cylinder050/wheel002" )[ 0 ];
+              }
+              return this.wheelFR;
+          wheelBR:
+            get: |
+              if ( !this.wheelBR ) {
+                this.wheelBR = this.find( "Cylinder014/Cylinder060/wheel003" )[ 0 ];
+              }
+              return this.wheelBR;
+          wheelRadius: 0.3304
+        children:
+          firstPerson:
+            extends: source/cameraMount.vwf
+            properties:
+              navmode: walk
+              translation: [ 0, 0, 1 ]
+              cameraPose: [ 0, 0, 0 ]
+              cameraSpeed: 0
+              targetVisible: false
+              useTargetRotation: true
+          thirdPerson:
+            extends: source/cameraMount.vwf
+            properties:
+              navmode: thirdPerson
+              cameraPose: [ 16, 0, -15 ]
+              cameraSpeed: 0
+          topDown:
+            extends: source/cameraMount.vwf
+            properties:
+              navmode: topDown
+              worldOffset: [ 0, 0, 40 ]
+              cameraPose: [ 0, 0, -90 ]
+              cameraSpeed: 10
+              cameraLock: false
+          Cylinder014:
+            extends: http://vwf.example.com/node3.vwf
+            children: 
+              Cylinder058:
+                extends: http://vwf.example.com/node3.vwf
+                children: 
+                  wheel:
+                    extends: http://vwf.example.com/node3.vwf
+              Cylinder059:
+                extends: http://vwf.example.com/node3.vwf
+                children: 
+                  wheel001:
+                    extends: http://vwf.example.com/node3.vwf
+              Cylinder050:
+                extends: http://vwf.example.com/node3.vwf
+                children: 
+                  wheel002:
+                    extends: http://vwf.example.com/node3.vwf
+                    children: 
+                      material:
+                        extends: http://vwf.example.com/material.vwf
+              Cylinder060:
+                extends: http://vwf.example.com/node3.vwf
+                children: 
+                  wheel003:
+                    extends: http://vwf.example.com/node3.vwf
+                    children: 
+                      material:
+                        extends: http://vwf.example.com/material.vwf
+          cargo:
+            extends: source/inventory.vwf
+            properties:
+              capacity: 4
+              inventoryIsVisible: false
+
```

Comment:
I don't know that we need to do this right now - but we really ought to move the rovers into their own files at some point (rather than having all of this buried in the scene).